### DISCUSSION TITLE
Add support for a much space and time efficient intermediate binary code representation, between dart and execution, like java's bytecode

### DISCUSSION THREAD

**Discussion by gintominto5329** [ethanblake4/dart_eval#93] [Ideas]
Hello,

Is it possible, that we give an option to the user to first compile(actually transpile) the dart code to an intermediate format, which is binary, and not human readable, which can be used at runtime, this should not be mandatory, but optional for extra perf, and efficiency

For example `switch` keyword takes 6+1 chars, but in binary, each keyword could have a uInt16 code, hence `switch` the keyword will take at-most 2 bytes of storage, this will also boost the speed of parsing,

@ethanblake4 
@fzyzcjy

thanks

**Comment by ethanblake4** [comment]
This library is already doing that :) it is called EVC bytecode, you can see all of the bytecodes [here](https://github.com/ethanblake4/dart_eval/blob/master/lib/src/eval/runtime/ops/all_ops.dart)

**Reply by SivaramSS** [reply]
Could you explain abbreviation of EVC & DBC?
